localization/english/buildings_l_english.yml

```diff
@@ -0,0 +1,1037 @@
+﻿l_english:
```

Comment:
You don't need to copy the entire file. Check `Warcraft-Guardians-of-Azeroth-2\localization\replace\english\wc_buildings_l_english.yml`